.github/workflows/CondaLock.yml

```diff
@@ -37,39 +37,37 @@ jobs:
       # Could run as single step in parallel, but would complicate logs...
       - name: Run conda-lock ${{ matrix.IMAGE }}
         run: |
-          cd ${{ matrix.IMAGE }}
-          if [ ${{ matrix.IMAGE }} = "base-notebook" ]; then
-            conda-lock lock -f environment.yml -p linux-64
-          elif [ ${{ matrix.IMAGE }} = "pangeo-notebook" ]; then
-            conda-lock lock -f environment.yml -f ../base-notebook/environment.yml -p linux-64
-          else
-            conda-lock lock -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-64
-          fi
-          conda-lock render -k explicit
-          ../generate-packages-list.py conda-linux-64.lock > packages.txt
-
-      - name: Upload lockfiles
-        uses: actions/upload-artifact@v4
-        with:
-          name: ${{ matrix.image }}
-          path: ${{ matrix.image }}
+          cd base-notebook
+          conda-lock lock -p linux-64 -p linux-aarch64 -p osx-64 -p osx-arm64
+	        conda-lock render -k explicit -p linux-64
+	        ../generate-packages-list.py conda-linux-64.lock > packages.txt
 
+      - name: Run conda-lock pangeo-notebook
+        timeout-minutes: 5
+        continue-on-error: true
+        run: |
+          cd pangeo-notebook
+          conda-lock lock -f environment.yml -f ../base-notebook/environment.yml -p linux-64 -p linux-aarch64 -p osx-64 -p osx-arm64
+          conda-lock render -k explicit -p linux-64
+          ../generate-packages-list.py conda-linux-64.lock > packages.txt
 
-  # Each job will commit files, so we know it succeeds based on commits
-  commit-lockfiles:
-    needs: condalock
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v4
-        with:
-          token: ${{ secrets.PANGEOBOT_TOKEN }}
-          # These lines are critical, otherwise Pangeo-bot pushes changes directly to master from PRs!
-          repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
-          ref: ${{ github.event.client_payload.pull_request.head.ref }}
+      - name: Run conda-lock ml-notebook
+        timeout-minutes: 5
+        continue-on-error: true
+        run: |
+          cd ml-notebook
+          conda-lock lock -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-64 -p linux-aarch64 -p osx-64 -p osx-arm64
+          conda-lock render -k explicit -p linux-64
+          ../generate-packages-list.py conda-linux-64.lock > packages.txt
 
-      # Download all artifacts from previous matrix job
-      - uses: actions/download-artifact@v4
+      - name: Run conda-lock pytorch-notebook
+        timeout-minutes: 5
+        continue-on-error: true
+        run: |
+          cd pytorch-notebook
+          conda-lock lock -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-64 -p linux-aarch64 -p osx-64 -p osx-arm64
```

Comment:
Pytorch does have linux-aarch64/osx-64/osx-arm64 packages on conda-forge, but we set a `cuda120` pin in the environment.yml file so locking fails:

```
Could not lock the environment for platform linux-aarch64
Could not solve for environment specs
The following packages are incompatible
├─ pytorch >=2.1.0 cuda120* is not installable because it requires
│  └─ __cuda, which is missing on the system;
└─ torchvision >=0.16.1 cuda120* does not exist (perhaps a typo or a missing channel).
```